02_Util/src/util/RequestOperations.ts

```diff
@@ -0,0 +1,22 @@
+import { IncomingMessage } from 'http';
+import { splitOnce } from './StringOperations';
+
+export function extractBasicCredentials(req: IncomingMessage): {
```

Comment:
We can probably then remove the private method in `WebsocketNetworkConnection:286` since this seems to duplicate.

Add some doc string pls.